<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite name="Failed suite [Suite]" guice-stage="DEVELOPMENT" verbose="0">
  <listeners>
    <listener class-name="FrameWork_TestClasses.testcomponant.Listeners"/>
  </listeners>
  <test thread-count="5" name="Test(failed)" verbose="0">
    <classes>
      <class name="FrameWork_TestClasses.tests.ErrorValidation">
        <methods>
          <include name="tearDown"/>
          <include name="loginErrorValidation"/>
          <include name="producterrorvalidation"/>
          <include name="launchApplication"/>
        </methods>
      </class> <!-- FrameWork_TestClasses.tests.ErrorValidation -->
      <class name="FrameWork_TestClasses.tests.SubmitOrder">
        <methods>
          <include name="submitOrder" invocation-numbers="1"/>
          <include name="tearDown"/>
          <include name="launchApplication"/>
          <include name="OrderhistoryTest"/>
        </methods>
      </class> <!-- FrameWork_TestClasses.tests.SubmitOrder -->
    </classes>
  </test> <!-- Test(failed) -->
</suite> <!-- Failed suite [Suite] -->
